nav2_bt_navigator/src/bt_navigator.cpp

```diff
@@ -55,8 +55,9 @@ BtNavigator::on_configure(const rclcpp_lifecycle::State & /*state*/)
   // Support for handling the topic-based goal pose from rviz
   client_node_ = std::make_shared<rclcpp::Node>("_", options);
 
-  self_client_ = rclcpp_action::create_client<nav2_msgs::action::NavigateToPose>(
-    client_node_, "navigate_to_pose");
+  self_client_ =
+    rclcpp_action::create_client<nav2_msgs::action::NavigateToPose>(client_node_,
+      "NavigateToPose");
```

Comment:
Should be snake case as @crdelsey mentions, `navigate_to_pose` What's publishing the wrong thing that needs to be updated?

Reply:
It should navigate to pose, but changing it to navigate_to_pose doesn't work. 
I think #1385 didn't cover all the necessary conversations in the action server names. For example [nav2_panel](https://github.com/ros-planning/navigation2/blob/f6defdbb70a2419e8585ac59ba09a0cdf1c1fe4e/nav2_rviz_plugins/src/nav2_panel.cpp#L228) still uses NavigationToPose.

Reply:
Ok, so an update to the panel is required? We shouldn't be changing the interface back

Reply:
using navigate_to_pose doesn't work. It seem like update is required in some other places as well. I am investigating this issue.

Reply:
@mlherd are you planning to pick this one up? If not, I'll merge it with #1451 and push that one instead.

Reply:
@hidmic Unfortunately, I won't have any time to work on this for next two weeks. Feel free work on #1451. Once your PR gets approved and merged, we can close this one.

Currently, #1451 is not a complete solution to this problem. Are you planning to do what @crdelsey sugested?

> change all the right places to navigate_to_pose from NavigateToPose

btw, thanks for contributing!

Reply:
@hidmic please move forward when you have the time, I'll close this PR if its not looking like it'll get in.